src/vs/platform/extensionManagement/common/extensionNls.ts

```diff
@@ -16,6 +16,7 @@ export function localizeManifest(extensionManifest: IExtensionManifest, translat
 	try {
 		replaceNLStrings(extensionManifest, translations, fallbackTranslations);
 	} catch (error) {
+		console.error(error?.message ?? error);
```

Comment:
Just checking that this is supposed to be left?

Reply:
Yes. Because I didn't have that, I didn't catch this issue, so I'm surfacing it